readme.md

```diff
@@ -131,7 +131,7 @@ Please feel free to submit bug reports on the github issue tracker at https://gi
 - SDL1 support removed d4da725
 - Dungeon level 2 generation #93
 
-##v0.1
+##v0.1 [24 Apr 2015]
```

Comment:
W dniu 23.11.2015 o 01:14, Robin Eklind pisze:
>> @@ -131,7 +131,7 @@ Please feel free to submit bug reports on the github issue tracker at https://gi
>>  - SDL1 support removed d4da725
>>  - Dungeon level 2 generation #93
>>  
>> -##v0.1
>> +##v0.1 [24 Apr 2015]
> 
> Should be `24 Apr 2014` rather than 2015?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/wheybags/freeablo/pull/177/files#r45567532
> 

Pardon, fixed.